src/main/java/org/launchcode/liftoffrecipetracker/Models/User.java

```diff
@@ -0,0 +1,56 @@
+package org.launchcode.liftoffrecipetracker.Models;
```

Comment:
When I opened this in Intellij, the IDE was confused about the package name because our models package is lowercase. If you lowercase the M it should fix it!